### PR TITLE
add explanation of blocking jobs in master before service streams

### PR DESCRIPTION
**Pull request by deads2k**
This isn't generally obvious, so it's worth explaining in the doc.  A linter would be even better, but I'm unsure of where such a linter would be.

### CODE REVIEW

File: core-services/release-controller/_releases/README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # Release Gating Jobs.
 
 This directory contains the configuration for release gating and informing jobs.  [Our documentation](https://docs.ci.openshift.org/docs/architecture/release-gating/) contains details on the future direction for nightly payload gating.  Please discuss with the [Technical Release Team approvers](/OWNERS_ALIASES) before working on any new blocking jobs.  Jobs marked as "optional: true" will be prioritized.
+
+Our strategy for ensuring backport quality and assessing backports for merge is based on acceptance and pass rates in master.
+Every backport is first merged into master, then passes automated testing, then can be backported.
+This means that a new job must first become blocking on master before becoming blocking on service
+streams in order to be confident that our backports will pass our service stream testing.
+If an exception is required due to instability of a job on master, that exception must be accepted
+by the party responsible for service stream quality, currently the rotation of patch managers.
```
Comment (sdodson) [inline core-services/release-controller/_releases/README.md:10]:
```suggestion
This means that a new job must first become blocking on master before becoming blocking on release
branches in order to be confident that our backports will pass our release branch testing.
If an exception is required due to instability of a job on master, that exception must be accepted
by the party responsible for release branch quality, currently the rotation of patch managers.
```
I feel like this is a more commonly understood term but maybe I'm wrong. Fine either way.

**Comment by sdodson** [comment]
/lgtm
/hold
Feel free to disregard my wording feedback or happy to lgtm after you've updated.

**Comment by deads2k** [comment]
updated word choice.

**Pull request merged**